src/constructs/lambda/lambda.ts

```diff
@@ -10,6 +10,14 @@ import type { GuStack } from "../core";
 import { AppIdentity } from "../core/identity";
 import { GuParameterStoreReadPolicyStatement } from "../iam";
 
+const DEPRECATED_RUNTIMES: Runtime[] = [
+  Runtime.NODEJS_4_3,
+  Runtime.NODEJS_6_10,
+  Runtime.NODEJS_8_10,
+  Runtime.NODEJS_10_X,
+  Runtime.NODEJS_12_X,
```

Comment:
IIUC, this [version](https://nodejs.org/en/about/releases/) becomes EOL on 2022-04-30 - do we want to deprecate it before then?

Reply:
Ah, I thought it was already deprecated! I think it's fine to do so before then as:
  - We're not blocking the synth step
  - It acts as a reminder to update _before_ the deadline
  - There aren't too many lambdas defined using GuCDK today (and 0 using this version of the library)